src/app-interoperability/application-package-definition.linkml.yaml

```
@@ -0,0 +1,497 @@
+          See the [Target](#target-attributes) section below.
+        rank: 30
+        range: Target
+        required: true
+        multivalued: true
+
+  Target:
+    description: Specifies where the parameter applies in the deployment.
+    rank: 100
+    attributes:
+      pointer:
+        description: >-
+          The name of the parameter in the deployment configuration. 
+          For Helm deployments, this is the dot notation for the matching element in the `values.yaml` file. This follows the same naming convention you would use with the `--set` command line argument with the `helm install` command. 
+          For docker-compose deployments, this is the name of the environment variable to set.
+        rank: 10
+        range: string
+        required: true
+      components:
+        description: >-
+          Indicates which deployment profile [component](#component-attribute) the parameter target applies to. 
+          The component name specified here MUST match a component name in the [deployment profiles](#deployment-profiles-attributes) section.
+        rank: 20
+        range: string
+        multivalued: true
+        required: true
+
+  Configuration:
+    description: Configuration layout and validation rules.
+    rank: 110
+    attributes:
+      sections:
+        description: >-
+          Sections are used to group related parameters together, 
+          so it is possible to present a user interface with a logical grouping of the parameters in each section. 
+          See the [Section](#section-attributes) section below.
+        rank: 10
+        range: Section
+        multivalued: true
+        required: true
+      schema:
+        description: >-
+          Schema is used to provide details about how to validate each parameter value. 
+          At a minimum, the parameter value must be validated to match the schema's data type. 
+          The schema indicates additional rules the provided value must satisfy to be considered valid input. 
+          See the [Schema](#schema-attributes) section below.
+        rank: 20
+        range: Schema
+        multivalued: true
+        required: true
+
+  Section:
+    description: Named sections within the configuration layout.
+    rank: 120
+    attributes:
+      name:
+        description: >-
+          The name of the section. This may be used in the user interface to show the grouping of the associated parameters within the section.
+        rank: 10
+        range: string
+        required: true
+      settings:
+        description: >-
+          Settings are used to provide instructions to the workload orchestration software vendor for displaying parameters to the user. 
+          A user MUST be able to provide values for all settings. 
+          See the [Setting](#setting-attributes) section below.
+        rank: 20
+        range: Setting
+        multivalued: true
```

@arne-broering `settings` is an array/list

```suggestion
        multivalued: true
        inlined: true
        inlined_as_list: true
```